tests/integration/customers.test.js

```diff
@@ -1634,14 +1634,53 @@ describe('CUSTOMERS QUOTES ROUTES', () => {
       expect(res.statusCode).toBe(400);
     });
 
+    it('should return a 400 error if \'service\' is missing from subscriptions in payload', async () => {
+      const payload = {
+        subscriptions: [
+          { unitTTCRate: 23, estimatedWeeklyVolume: 3 },
+          { service: { name: 'TestTest', nature: 'hourly' }, unitTTCRate: 30, estimatedWeeklyVolume: 10 },
+        ],
+      };
+
+      const res = await app.inject({
+        method: 'POST',
+        url: `/customers/${customersList[1]._id.toHexString()}/quotes`,
+        payload,
+        headers: { Cookie: `alenvi_token=${authToken}` },
+      });
+
+      expect(res.statusCode).toBe(400);
+    });
+
+    const params = [{ name: 'TestTest' }, { nature: 'hourly' }];
+    params.forEach((service) => {
+      it(`should return a 400 error if ${Object.keys(service)} is missing from subscriptions in payload`, async () => {
```

Comment:
j'ai l'impression que tu as codé a l'envers : 
pour le test `should return a 400 error if name is missing from subscriptions in payload` c'est `nature` qui est manquant

Reply:
done